features/kafka/producer/src/main/java/org/opennms/features/kafka/producer/OpennmsKafkaProducer.java

```diff
@@ -134,12 +138,31 @@ private void forwardEvent(Event event) {
         if (forwardNodes && event.getNodeid() != null && event.getNodeid() != 0) {
             maybeUpdateNode(event.getNodeid());
         }
-        sendRecord(() -> {
 
-            final OpennmsModelProtos.Event mappedEvent = protobufMapper.toEvent(event).build();
-            LOG.debug("Sending event with UEI: {}", mappedEvent.getUei());
-            return new ProducerRecord<>(eventTopic, mappedEvent.getUei(), mappedEvent.toByteArray());
-        });
+        boolean result = true;
```

Comment:
Can we rename this variable? Something like `shouldForwardEvent` is a lot more indicative than`result`.